rocketpy/Components.py

```diff
@@ -0,0 +1,125 @@
+__author__ = "Mateus Stano Junqueira"
+__copyright__ = "Copyright 20XX, RocketPy Team"
+__license__ = "MIT"
+
+from collections import namedtuple
+
+
+class Components:
+    """A class to hold components of the Rocket class. A component is
+    an object that added to the rocket given a position relative to the
+    rocket's coordinate system origin. Components can be added to the
+    rocket using the 'add' methods. This class is currently used only to
+    hold aerodynamic surfaces.
+
+    Attributes
+    ----------
+    _components: list
+        A list of named tuples containing the component and its position.
+    component_tuple: namedtuple
+        A named tuple containing the component and its position.
+    """
+
+    def __init__(self):
+        """Initialize an empty components list instance."""
+        self.component_tuple = namedtuple("component_tuple", "component position")
+        self._components = []
+
+    def __repr__(self):
+        """Return a string representation of the Components instance."""
+        return repr(self._components)
+
+    def __len__(self):
+        """Return the number of components in the collection."""
```

Comment:
In the class docstring you cite this as a list, but from now on you mention as a collection.

What is the point? Is it a list or a collection afterwards?